### PR TITLE
Added CVE-2021-41773

### PR DESCRIPTION
**Pull request by realmatthewpeng**
This CVE is in CISA's Known Exploited Vulnerabilities Catalog: [CISA Link](https://www.cisa.gov/known-exploited-vulnerabilities-catalog?search_api_fulltext=CVE-2021-41773&field_date_added_wrapper=all&field_cve=&sort_by=field_date_added&items_per_page=20&url=)

### CODE REVIEW

File: CVE-2021-41773/cvex.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+blueprint: ubuntu2204-ubuntu2204
+ubuntu1:
+  playbook: ubuntu1.yml
+ubuntu2:
+  playbook: ubuntu2.yml
+  command: "python3 /opt/exploit.py -u http://192.168.56.3:80"
```
Comment (olegbck) [inline CVE-2021-41773/cvex.yml:6]: Please use %ubuntu1% instead of 192.168.56.3, do not hardcode IP addresses

Comment (realmatthewpeng) [reply CVE-2021-41773/cvex.yml]: Updated

**Comment by olegbck** [comment]
Please remove output.txt and then I'll merge it

**Comment by realmatthewpeng** [comment]
@olegbck Deleted output.txt

**Pull request merged**